File: Java/01.ActiveMq.NoVerify/context-shared/src/main/java/org/shared/Tasks/TaskCreated.java
package org.shared.Tasks;


import java.util.Date;
import java.util.UUID;
import org.cqrs.Event;

public class TaskCreated implements Event {

    private UUID id;
    private Date creationDate;
    private TaskDueDateChanged dueDateSet;
    private TaskPriorityChanged prioritySet;
    private TaskDescriptionChanged descriptionSet;
    private TaskTitleChanged titleSet;

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public Date getCreationDate() {
        return creationDate;
    }

    public void setCreationDate(Date creationDate) {
        this.creationDate = creationDate;
    }

    public TaskDueDateChanged getDueDateSet() {
        return dueDateSet;
    }

    public void setDueDateSet(TaskDueDateChanged dueDateSet) {
        this.dueDateSet = dueDateSet;
    }

    public TaskPriorityChanged getPrioritySet() {
        return prioritySet;
    }

    public void setPrioritySet(TaskPriorityChanged prioritySet) {
        this.prioritySet = prioritySet;
    }

    public TaskDescriptionChanged getDescriptionSet() {
        return descriptionSet;
    }

    public void setDescriptionSet(TaskDescriptionChanged descriptionSet) {
        this.descriptionSet = descriptionSet;
    }

    public TaskTitleChanged getTitleSet() {
        return titleSet;
    }

    public void setTitleSet(TaskTitleChanged titleSet) {
        this.titleSet = titleSet;
    }
}

File: Net/Step0/Commons.Domain/Tasks/TaskCreated.cs
using Cqrs;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Commons.Tasks
{
    public class TaskCreated : IEvent
    {
        public TaskCreated()
        {

        }
        public Guid Id { get; set; }
        public DateTime CreationDate { get; set; }
        public TaskDueDateChanged DueDateSet { get; set; }
        public TaskPriorityChanged PrioritySet { get; set; }
        public TaskDescriptionChanged DescriptionSet { get; set; }
        public TaskTitleChanged TitleSet { get; set; }
    }
}
